spec/example_shrinking_spec.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+RSpec.describe 'shrinking' do
+  include Hypothesis::Debug
+
+  it 'finds lower bounds on integers' do
+    n, = find { given(integers) >= 10 }
```

Comment:
Oh wow this syntax looks super nice

Reply:
Yeah, there's a lot I'm ambivalent about in this implementation but I'm really pleased with the syntax. It's both a refreshed Hypothesis-how-I-would-do-it-from-scratch and Ruby's syntactic flexibility, for all its faults, works out really well here.